### PR TITLE
First-Capstone-project

### PR DESCRIPTION
**Pull request by FaranoshAmini**
In this capstone project, I build a website for Microverse by following the design guidelines including Color, Typograpghy, and Layout from the original design on Behance by [Cindy Shin](https://www.behance.net/adagio07). During this project, I designed two different pages

Home Page
About Page
I designed these two pages for both mobile and desktop users.

### CODE REVIEW

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @FaranoshAmini :wave: , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't fulfill the following:

- [ ] You should fix the failing linters before submitting you pull request for review.
image

- [ ] Only one `linters.yml` file is enough for the Github Actions workflow. would you please elaborate when two are needed in your case?
image

- [ ] You should have a professional ReadMe for every project you do. It is also suggested to use this [ReadMe template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse.
image

- [ ] I couldn't find video presentation of the project. Since it part of the project requirement, kindly add that as well.
image

- [ ] Kindly use a descriptive pull request summary and pull request title
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+# MICROVERSE
+
+> This  Web page is  about MICROVERSE that contains information about Main program. I have used HTML, CSS and JavaScript for this project and you can use it in your local machin just by cloning this repo.
+
+
+## 🖥️ Live Demo
+
+-[SHORT DESCRIPTION](https://www.loom.com/share/1f1534db165e41c4bbc3bf05fb467947)
+
+-[LIVE DEMO](https://faranoshamini.github.io/First-Capstone-Project/)
```
Comment (sja-thedude) [inline README.md:10]: - [ ] Kindly make sure to deploy your website correctly, currently, this link is not displaying anything.

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Special Thanks [Cindy Shin](https://www.behance.net/adagio07)
```
Comment (sja-thedude) [inline README.md]: - [ ] Kindly repharse this as "I would like to give credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the Creative Commons License."

File: about.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+      </nav>
+    </header>
+
+    <!--start of home page-->
+    <section class="heading" id="home-1">
+      <div class="home">
+        <div class="first"> <img src="Images\hamburger.png" alt="Humbergar
+            icon" class="hamburger" width="50px" id="hamburger">
+        </div>
+        <div class="sub-title">What Makes Microverse Different?</div>
```
Comment (sja-thedude) [inline about.html]: - [ ] Kindly wrap the heading in h1 tags.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+  More: 'Emilia is a web developer by profession and a teacher at heart.',
+},
+{
+  Name: 'Gaby Suarez',
+  Title: 'ADMISSIONS',
+  Image: 'images/Gaby.png',
+  More: 'Gaby is a native Salvadoran, Spanglish-speaking, dog (and sushi) lover.',
+},
+];
+
```
Comment (sja-thedude) [inline script.js]: - [ ] Kindly make sure to add 2 more speakers as suggested in the attached loom video. 
- [ ] Kindly make sure to add background checks image to all the images as suggested in the attached loom video. 
- [ ] Kindly make sure to add more button for the mobile version as suggested in the attached loom video.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+                    <p class="sub-description">A website to host your
+                        repositories online.</p>
+                </div>
+                <div class="sub-section">
+                    <img class="sub-image" src="Images/css.png" alt=""
+                        id="logo-section">
+                    <h4 class="section-title">CSS</h4>
+                    <p class="sub-description">CSS is a language that describes
+                        the style of an HTML document.</p><br>
+                </div>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly make sure to add 2 more cards as suggested in the attached loom video. 
- [ ] Kindly make sure to follow the spacing as suggested in the attached loom video.

File: about.html [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+          <ul class="main">
+            <li class="menu"><a href="#main-pro"
+                class="menu" id="program">Program</a></li>
+            <li class="menu"><a href=""
+                class="menu" id="speaker">Speakers</a></li>
+            <li class="menu"><a href="#partner"
+                class="menu" id="spoLOnsor">Sponsors</a></li>
+            <li class="menu"><a href="about.html"
+                class="menu" id="openAbout">About</a></li>
```
Comment (sja-thedude) [inline about.html]: - [ ] Kindly make sure to replace about with home as suggested in the loom video.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @FaranoshAmini, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ All linters passing
✔️ Good job so far

## Required 
- Video presentation:
- [ ] Kindly record another video highlighting a piece of the code the student finds interesting or is proud of making in the video.
- [ ] Make sure your face is visible while recording the video presentation as it's a requirement.

- Capstone
- [ ] Kindly view [this](https://www.loom.com/share/309d61341a2b4b3798c2f7fc4dd92e4b) loom video to make changes.
- [ ] Kindly make sure to follow the spacing, width, height, and font size as suggested in the loom video.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @FaranoshAmini 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by FaranoshAmini** [comment]
@omarsalem7 Thank you so much for your kind review and feedback.

**Pull request merged**